components/widgets/loading/loading_wrapper.test.tsx

```diff
@@ -47,7 +47,6 @@ describe('components/widgets/loading/LoadingWrapper', () => {
                 <LoadingWrapper
                     loading={testCase.loading}
                     text={testCase.text}
-                    type={testCase.type}
                 >
```

Comment:
why no 'type' ?

Reply:
We are no longer having "type", we used to have "spinner" loading and "bars" loading, but the bars loading no longer exists, so everything is a spinner.